docs/introduction.rst

```diff
@@ -19,42 +31,82 @@ can be open source or proprietary (although some distribution services, like
 `Flathub <https://flathub.org/>`_, can have restrictions in this respect).
 
 The only technical requirements made by Flatpak are that applications follow a
-small number of Freedesktop standards, in order to enable desktop integration
+small number of FreeDesktop standards, in order to enable desktop integration
 (see :doc:`conventions`).
 
+Issues of current model of packaging
+------------------------------------
+
+It is important to understand the problems of the current model
+of packaging applications to understand the existence of Flatpak:
+
+- **Duplicated work packaging apps**: Many Linux distributions come with their own package
+  manager, package format and repository. This requires a lot of maintainers to package the
+  same application in various distributions, and/or the application developer to learn the
+  language of each format and then package the application in those distributions, and/or
+  ignore most distributions and package and support a couple of distributions. This makes
+  it very difficult to innovate the Linux desktop as a whole as a considerably high amount of time
```

Comment:
I think this needs rephrasing. Application developers aren't necessarily trying to "innovate the desktop as a whole". You have already talked about the duplicated work; perhaps just say this makes the Linux desktop a difficult platform for software vendors to target?